invenio_vocabularies/contrib/names/config.py

```diff
@@ -28,9 +28,9 @@ class NamesSearchOptions(SearchOptions):
 
     suggest_parser_cls = SuggestQueryParser.factory(
         fields=[
-            "name^100",
-            "family_name^100",
-            "given_name^100",
+            "name^80",
```

Comment:
shouldn't these also utilize the `search_as_you_type` suggest field as well?

Reply:
Yes, added, thanks!